src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -61,20 +61,20 @@ public EditCommand parse(String args) throws ParseException {
             editPatientDescriptor.setPhone(ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get()));
         }
         if (argMultimap.getValue(PREFIX_EMAIL).isPresent()) {
-            editPatientDescriptor.setEmail(ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get()));
+            editPatientDescriptor.setEmail(ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL)));
```

Comment:
You could maybe use the `ifPresent` method

Reply:
Can u elaborate a little? Because I don't what you're talking about